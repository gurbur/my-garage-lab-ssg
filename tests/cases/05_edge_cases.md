# 엣지 케이스 테스트

리스트 항목 안에 코드 블럭이 포함된 경우:
- 첫 번째 항목
- 두 번째 항목
    ```c
    // 리스트 항목 안의 코드 블록
    for (int i = 0; i < 5; i++) {
        //...
    }
    ```
- 세 번째 항목

* * *

수평선처럼 보이지만, 뒤에 글자가 와서 수평선이 아닌 경우:
--- 이것은 수평선이 아닙니다.

잘못된 강조 구문: *이탤릭이
**볼드가 닫히지 않았습니다.
