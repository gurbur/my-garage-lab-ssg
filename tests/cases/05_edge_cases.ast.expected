-> DOCUMENT
	-> HEADING1 | data1: "엣지 케이스 테스트"
	-> PARAGRAPH
		-> TEXT | data1: "리스트 항목 안에 코드 블럭이 포함된 경우: - 첫 번째 항목 - 두 번째 항목     "
		-> CODE | data1: ""
		-> TEXT | data1: "`c     // 리스트 항목 안의 코드 블록
    for (int i = 0; i < 5; i++) {
        //...
    }
    "
		-> CODE | data1: ""
		-> TEXT | data1: "` - 세 번째 항목"
	-> UNORDERED_LIST
		-> LIST_ITEM
			-> TEXT | data1: " "
			-> ITALIC | data1: " "
	-> PARAGRAPH
		-> TEXT | data1: "수평선처럼 보이지만, 뒤에 글자가 와서 수평선이 아닌 경우: --- 이것은 수평선이 아닙니다."
	-> PARAGRAPH
		-> TEXT | data1: "잘못된 강조 구문: *이탤릭이 **볼드가 닫히지 않았습니다. "
